### ISSUE TITLE
AttributeError: 'GFile' object has no attribute 'tell'

### ISSUE THREAD

**Issue by mfurlend** [ryanwebber/tensorflow-image-classification#1]
I am attempting to run this code. I have tried it on a regular Python 2.7 installation (OSX), as well as with a `docker` image (gcr.io/tensorflow/tensorflow) based on Ubuntu.

I set up the training/testing images, and successfully ran the `built_data_set` script, which generated the gzipped files. However, when I attempt to run `python classify.py` I get this error: 

```
root@5a4f34840824:/notebooks/tensorflow-image-classification# python classify.py
Extracting data/output/TFtrain.images.gz
Traceback (most recent call last):
  File "classify.py", line 229, in <module>
    tf.app.run()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/platform/app.py", line 30, in run
    sys.exit(main(sys.argv[:1] + flags_passthrough))
  File "classify.py", line 225, in main
    run_training()
  File "classify.py", line 131, in run_training
    data_sets = tf_data.read_data_sets(FLAGS.train_dir, FLAGS.fake_data)
  File "/notebooks/tensorflow-image-classification/tf_data.py", line 181, in read_data_sets
    train_images = extract_images(local_file)
  File "/notebooks/tensorflow-image-classification/tf_data.py", line 41, in extract_images
    magic = _read32(bytestream)
  File "/notebooks/tensorflow-image-classification/tf_data.py", line 34, in _read32
    return numpy.frombuffer(bytestream.read(4), dtype=dt)[0]
  File "/usr/lib/python2.7/gzip.py", line 261, in read
    self._read(readsize)
  File "/usr/lib/python2.7/gzip.py", line 288, in _read
    pos = self.fileobj.tell()   # Save current position
AttributeError: 'GFile' object has no attribute 'tell'
```

This error seems to be related to this blog post: 
http://www.enricozini.org/blog/2011/cazzeggio/python-gzip/

I'd appreciate some help getting around this issue.
Thanks!

**Comment by ryanwebber** [comment]
Have you had any progress into solving this?

**Comment by popivol** [comment]
i have the same problem as above, have u found a solution to it?

**Comment by ryanwebber** [comment]
I'll take a look

**Comment by ryanwebber** [comment]
This should be fixed now with b1bee24. 

Note that a few of the functions used for training and learning are depreciated, so for now, you might be best to use examples of the tensorflow website, although the image and label generation should be still useful. I'll make an issue to update this, and clean up the repo.
